README.md

```diff
@@ -550,6 +550,32 @@ console.log(newName); // ['Ryan', 'McDermott'];
 ```
 **[⬆ back to top](#table-of-contents)**
 
+### Avoid Side Effects pt.2
+Side effects could also occur from inside a function. In Javascript, function arguments 
+are always passed by value except when they(functions) are passed reference values such as 
```

Comment:
We can reword this sentence. We are saying here "JavaScript always does this, except when it doesn't." Many guides word it this way, but it's kind of a pedantic way of saying it. Yes technically that's _true_, you are passing a value in every case, but if your value is a pointer then it's a reference. Anyway, we can just say "In JavaScript, primitives are passed by value and objects are passed by reference."